### DISCUSSION TITLE
Any updates on Multiselect and Combobox ?

### DISCUSSION THREAD

**Discussion by Thesmit113** [tremorlabs/tremor#63] [General]
wanted to know if there are any updates on when these components might be available.

**Comment by Thesmit113** [comment]
It seems they don't have any plans for this at the moment. Previously, it was listed on the website as "coming soon," but now it has been removed. Could you guys, from Tremor, please provide any updates on this?

**Reply by vasinl124** [reply]
+1

**Reply by lightsound** [reply]
+1
